platform-ui-skin/src/main/webapp/skin/less/variables.less

```diff
@@ -41,6 +41,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 @primaryBackgroundDefault:   #f0f0f0;
 @primaryBackground:   var(--allPagesPrimaryBackground, @primaryBackgroundDefault); // main background such as , header popup background, tab items...
 
+// Background Disable Color
+@disabledBackgroundButton: #e0e0e0;
```

Comment:
Put this value in a css var ==> var(--name, value)
**Example:** 
@disabledBackgroundButtonDefault: #e0e0e0;
@primaryBackground: var(--allPagesDisabledBackgroundButton, @disabledBackgroundButtonDefault);